src/tex/ms.tex

```diff
@@ -196,11 +196,11 @@ \subsection{Model evaluation}
 
 \paragraph{Judging chemical preference}
 
-One interesting finding of recent research is that foundation models can judge interestingness.\autocite{zhang2024omniopenendednessmodelshuman}
+One interesting finding of recent research is that foundation models can judge interestingness.\autocite{zhang2024omniopenendednessmodelshuman, Argyle_2023}
 If models could do so for chemical compounds, this would open opportunities for novel optimization approaches.
 Such open-ended tasks, however, depend on an external observer defining what interestingness is.\autocite{hughes2024openendednessessentialartificialsuperhuman}
 Here, we posed models the same question~\textcite{Choung_2023} asked chemists at a drug company: \enquote{Which of the two compounds do you prefer?} (in the context of an early virtual screening campaign setting).
```

Comment:
Maybe add a reference to the chembench examples table?

Reply:
yep - do you want to add a suggested change we can directly commit here?